### ISSUE TITLE
Out of memory error with Docs tests

### ISSUE THREAD

**Issue by simone-silvestri** [CliMA/Oceananigans.jl#3779]
For example:
https://buildkite.com/clima/oceananigans/builds/17468#0191fc2c-b421-4cf7-80b5-0429336b1d7f
https://buildkite.com/clima/oceananigans/builds/17473#0191fd88-d8b9-48d5-9c7f-18efc6747ea7
I believe this is because we are launching the docs from many different branches on a relatively small GPU. I think it would be best to move this test on the caltech cluster (I can open a PR to fix this)

**Comment by glwagner** [comment]
No, that's not the reason and I don't think we should move the docs to the caltech cluster. The reason is that other users of tartarus are using GPU 0.

**Comment by glwagner** [comment]
I mean, we can move the docs to the caltech cluster but I think they will slow down a lot. This is a bottleneck for us right now so I don't think we can afford to move them...

Notice that the out of memory error doesn't occur when we are using the GPU. We only use the GPU for the one quick start example --- and for nothing else.

If we want to "solve" this, we can just get rid of the quick start example and then return to the previous behavior where we set `CUDA_VISIBLE_DEVICES=-1` for the docs build.

Another solution is to hide / prevent tartarus users from using GPU 0.

**Comment by glwagner** [comment]
I think we can hide GPU 0

**Comment by glwagner** [comment]
I could see us eventually moving towards using tartarus only for docs. We could do some bigger GPU stuff then.

**Comment by glwagner** [comment]
Just take a step back. We had a working system until we exposed the GPU. I did that as an experiment and added the quick start example.

Now, if the experiment isn't working, let's revisit it. Moving the docs to caltech is a nuclear option. If it gives us speed up --- great. That's a good reason. But if it's just for the GPU issue, it makes no sense. It's like we tried to experiment with a new vegetable in our pasta sauce, didn't like the vegetable, and decided to stop eating dinner altogether as as result. It's not logical.

**Comment by simone-silvestri** [comment]
Ok I am convinced, I will close the PR and maybe we can monitor more closely GPU 0 for the moment until we find a stable solution.

**Comment by glwagner** [comment]
Should we set `CUDA_VISIBLE_DEVICES=-1` back for the docs? We just have to change the GPU quick start which is not a big deal (changing from a doctest to a static example)

**Comment by glwagner** [comment]
This is an old issue so I am assuming is closed
